webapp/src/components/sidebar_right/sidebar_right.jsx

```diff
@@ -111,27 +111,27 @@ export default class SidebarRight extends React.PureComponent {
         switch (this.props.rhsState) {
         case RHSStates.PRS:
 
-            githubItems = this.props.yourPrs;
+            githubItems = this.props.yourPrs || [];
```

Comment:
If we want to do this this way, we should have the empty array defined as a constant, so we don't change the reference to the `githubItems` prop being passed to the child component on every render.

```suggestion
            githubItems = this.props.yourPrs || emptyArray;
```